Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -732,8 +732,19 @@ Optional<ARIA::Role> HTMLElement::default_role() const
     if (local_name() == TagNames::article)
         return ARIA::Role::article;
     // https://www.w3.org/TR/html-aria/#el-aside
-    if (local_name() == TagNames::aside)
+    if (local_name() == TagNames::aside) {
+        // https://w3c.github.io/html-aam/#el-aside
+        for (auto const* ancestor = parent_element(); ancestor; ancestor = ancestor->parent_element()) {
+            if ((ancestor->local_name() == TagNames::article
+                    || ancestor->local_name() == TagNames::aside
+                    || ancestor->local_name() == TagNames::nav
+                    || ancestor->local_name() == TagNames::section)
```

Comment:
You can make this a bit more succinct with `first_is_one_of(ancestor->local_name(), TagNames::article, TagNames::aside, TagNames::main, TagNames::nav, TagNames::section)`.

Reply:
> You can make this a bit more succinct with `first_is_one_of(ancestor->local_name(), TagNames::article, TagNames::aside, TagNames::main, TagNames::nav, TagNames::section)`.

Thanks — made it so

Reply:
This doesn't appear to have changed. Is there a change that needs to be pushed?

Reply:
oops, thanks — there were two instances that needed replacement with `first_is_one_of`, but I had only replaced one